src/materialized/src/bin/materialized/main.rs

```diff
@@ -499,7 +499,13 @@ swap: {swap_total}KB total, {swap_used}KB used",
         dep_versions = build_info().join("\n"),
         invocation = {
             use shell_words::quote as escape;
-            env::vars()
+            env::vars_os()
+                .map(|(name, value)| {
+                    (
+                        name.to_string_lossy().into_owned(),
+                        value.to_string_lossy().into_owned(),
+                    )
+                })
```

Comment:
out of curiosity, why did this need to change?

Reply:
It’s in the message for the other commit!